app/Console/Commands/CreateDatabase.php

```diff
@@ -39,67 +40,75 @@ public function __construct()
+    	}
     }
 
     /**
-     * Function to create databse. If database name is not specified in .env file, it will display the messasge and exits.Then it checks if database already exists
+     * Function to create databases. If database name is not specified in .env file,
+     * it will display the messasge and exits.Then it checks if database already exists
+     * If not, it will create database for it. It's used to create 2 dbs in handle method.
+     * @param db_name Database name
+     * @param db_host Database hostname
+     * @param db_port Database port number
+     * @param db_username Database username
+     * @param db_password Database user's password
+     * @return void
+     * @author 2018 Tigpezeghe Rodrige K. <tigrodrige@gmail.com>
+     */
+    private function createDB($db_name, $db_host, $db_port, $db_username, $db_password)
+    {
+        /* LibreLaravel database creation logic. */
+        if(!$db_name) {
+            $this->info('Database name is not specified in .env file. Please check that both database names are specified.');
+        } else {
+            $pdo_instance = $this->getPDO($db_host, $db_port, $db_username, $db_password);
+            $check_db_exists = $pdo_instance->prepare('select count(*) from INFORMATION_SCHEMA.SCHEMATA where SCHEMA_NAME = :dbname');
+            $check_db_exists->bindParam(':dbname', $db_name, PDO::PARAM_STR, 12);
+            $check_db_exists->execute();
+            $check_db_exists->setFetchMode(PDO::FETCH_ASSOC);
+            /*If database exist it will return 1 else 0 in $check_db_exists variable. */
+            if(!$check_db_exists->fetchColumn()) {
+                $pdo_instance->exec(sprintf('CREATE DATABASE IF NOT EXISTS %s;', $db_name));
+                $this->info(sprintf('Created Database %s Successfully. Run migrations to install tables.', $db_name));
+            } else {
+                $this->info(sprintf('Database %s already exists. Migrate the tables.', $db_name));
+            }
+        }
+    }
+
+    /**
+     * Function to create databases. If database name is not specified in .env file, it will display the messasge and exits.Then it checks if database already exists
      * If not then it will create database for it.
      * @param None
-     * @return void 
+     * @return void
      */
-
     public function handle()
     {
-	/* Take the database configuration from .env file. */
-        $databaseName = env('DB_DATABASE', false);
-	$databaseHost = env('DB_HOST');
-	$databasePort = env('DB_PORT');
-	$databaseUser = env('DB_USERNAME');
-	$databasePassword = env('DB_PASSWORD');
+        /** librelaravel database creation logic.
+         *  @TODO: Uncomment the line below when EHR is in full Laravel mode.
+         *  That is, when the EHR has been completely ported to Laravel.
+         */
+    	 //$this->createDB(env('DB_DATABASE'), env('DB_HOST'), env('DB_PORT'), env('DB_USERNAME'), env('DB_PASSWORD'));
```

Comment:
I have uncommented this line to make it working.

Reply:
Should I uncomment and push?